cfgov/unprocessed/css/molecules/multi-select.less

```diff
@@ -80,27 +80,30 @@
     }
 
     &_choices {
+        margin-right: unit( -10px / @base-font-size-px, em );
         margin-bottom: 0;
 
         li {
             display: inline-block;
 
-            margin-right: unit( 15px / @base-font-size-px, em );
+            margin-right: unit( 10px / @base-font-size-px, em );
         }
 
         li:last-child {
-          margin-bottom: unit( 5px / @base-font-size-px, em );
+          margin-bottom: unit( 10px / @base-font-size-px, em );
         }
 
         label {
-            display: block;
+            display: inline-block;
             padding: 2px 5px;
+            padding-right: unit( 25px / 14px, em );
```

Comment:
Use `@size-v` here in place of the 14px. Also on L:102 (border radius is incorrectly calculating ems based on 16px) in place of `@base-font-size-px`.